changelogs/fragments/280-vlan_expander.yaml

```
@@ -0,0 +1,3 @@
+---
+major_changes:
```

```suggestion
minor_changes:
```
Major changes are backwards incompatible changes to existing features, which this is not

---

Thank you.  I committed it.